src/libstd/ffi/c_str.rs

```diff
@@ -1040,7 +1040,8 @@ impl CStr {
     /// ```
     #[inline]
     #[stable(feature = "cstr_from_bytes", since = "1.10.0")]
-    pub unsafe fn from_bytes_with_nul_unchecked(bytes: &[u8]) -> &CStr {
+    #[rustc_const_unstable]
+    pub const unsafe fn from_bytes_with_nul_unchecked(bytes: &[u8]) -> &CStr {
```

Comment:
@oli-obk preferred ordering of the attributes? shortest to longest or doesn't really matter?